submissions/adeolaFafemi/html-forms/contactUs.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Under9k Store</title>
+</head>
+<body>
+    <img src="../html/images/image1.jpg" alt="logo" height="100">
+    <h3>Under 9k store</h3>
+    <section>
+        <nav>
+            <aside>
+                <ul>
+                    <li><a href="../htmlForms/signUp.html">Sign Up</a></li>
```

Comment:
There is another task to link your pages. You should remove this and add it for that task @kojusola

Reply:
Removed the links in the contact us page.
Thank you.
![contactUs2](https://user-images.githubusercontent.com/66546900/94671734-27f83180-030c-11eb-98ed-97fb0eda93f1.jpg)
![contactUs](https://user-images.githubusercontent.com/66546900/94671788-3a726b00-030c-11eb-8dd4-ee7316787f9a.jpg)